cypress/e2e/20-workflow-executions.cy.ts

```diff
@@ -289,15 +289,18 @@ describe('Workflow Executions', () => {
 });
 
 const createMockExecutions = () => {
-	executionsTab.actions.createManualExecutions(5);
+	executionsTab.actions.createManualExecutions(15);
 	// Make some failed executions by enabling Code node with syntax error
+	cy.wait(2000);
```

Comment:
This was added to allow time for the notifications to expire. We should be able to remove this in the future when the notification handling is better.

Reply:
Can yo add this as a comment here and below? So that it's not los.